flink-doris-connector/src/main/java/org/apache/doris/flink/sink/schema/SQLParserSchemaManager.java

```diff
@@ -0,0 +1,191 @@
+                            sourceConnector, datatype, length, scale);
+
+            List<String> columnSpecs = columnDataType.getColumnSpecs();
+            String defaultValue = extractDefaultValue(columnSpecs);
+            String comment = extractComment(columnSpecs);
+            FieldSchema fieldSchema = new FieldSchema(columnName, datatype, defaultValue, comment);
+            String addColumnDDL = SchemaChangeHelper.buildAddColumnDDL(dorisTable, fieldSchema);
+            LOG.info("Parsed add column DDL SQL is: {}", addColumnDDL);
+            addColumnList.add(addColumnDDL);
+        }
+        return addColumnList;
+    }
+
+    private String processChangeColumnOperation(
+            AlterExpression alterExpression, String dorisTable) {
+        String columnNewName = alterExpression.getColDataTypeList().get(0).getColumnName();
+        String columnOldName = alterExpression.getColumnOldName();
+        String renameColumnDDL =
+                SchemaChangeHelper.buildRenameColumnDDL(dorisTable, columnOldName, columnNewName);
+        LOG.warn(
+                "Note: Only rename column names are supported in doris. "
+                        + "Therefore, the change syntax used here only supports the use of rename."
+                        + " Parsed change column DDL SQL is: {}",
+                renameColumnDDL);
+        return renameColumnDDL;
+    }
+
+    private String processRenameColumnOperation(
+            AlterExpression alterExpression, String dorisTable) {
+        String columnNewName = alterExpression.getColumnName();
+        String columnOldName = alterExpression.getColumnOldName();
+        String renameColumnDDL =
+                SchemaChangeHelper.buildRenameColumnDDL(dorisTable, columnOldName, columnNewName);
+        LOG.info("Parsed rename column DDL SQL is: {}", renameColumnDDL);
+        return renameColumnDDL;
+    }
+
+    private String extractDefaultValue(List<String> columnSpecs) {
+        String defaultValue = null;
+        if (columnSpecs.contains("default")) {
+            int defaultIndex = columnSpecs.indexOf("default");
+            defaultValue = removeQuotes(columnSpecs.get(defaultIndex + 1));
+        } else if (columnSpecs.contains("DEFAULT")) {
+            int defaultIndex = columnSpecs.indexOf("DEFAULT");
+            defaultValue = removeQuotes(columnSpecs.get(defaultIndex + 1));
+        }
+        return defaultValue;
+    }
+
+    private String extractComment(List<String> columnSpecs) {
+        String comment = null;
+        if (columnSpecs.contains("comment")) {
+            int commentIndex = columnSpecs.indexOf("comment");
+            comment = removeQuotes(columnSpecs.get(commentIndex + 1));
+        }
+        if (columnSpecs.contains("COMMENT")) {
+            int commentIndex = columnSpecs.indexOf("COMMENT");
+            comment = removeQuotes(columnSpecs.get(commentIndex + 1));
+        }
+        return comment;
+    }
```

Comment:
same above, does JSqlParser have any suspicious methods to directly obtain the Default value and Comment?